cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -561,6 +567,48 @@ func (s *Supervisor) loadAgentEffectiveConfig() {
 	}
 
 	s.effectiveConfig.Store(string(effectiveConfigBytes))
+
+	if s.config.Capabilities != nil && s.config.Capabilities.AcceptsRemoteConfig != nil &&
+		*s.config.Capabilities.AcceptsRemoteConfig &&
+		s.config.Storage != nil {
+		// Try to load the last received remote config if it exists.
+		lastRecvRemoteConfig, err = os.ReadFile(filepath.Join(s.config.Storage.Directory, lastRecvRemoteConfigFile))
+		if err == nil {
+			config := &protobufs.AgentRemoteConfig{}
+			err = proto.Unmarshal(lastRecvRemoteConfig, config)
+			if err != nil {
+				s.logger.Error("Cannot parse last received remote config", zap.Error(err))
+			} else {
+				s.remoteConfig = config
+			}
+		}
+	} else {
+		s.logger.Debug("Remote config is not supported")
```

Comment:
```suggestion
		s.logger.Debug("Remote config is not supported, will not attempt to load config from file")
```
We should probably be more specific about the context of the log.